cmd/csi-node-driver-registrar/main.go

```diff
@@ -99,9 +101,18 @@ func (e registrationServer) NotifyRegistrationStatus(ctx context.Context, status
 }
 
 func main() {
-	klog.InitFlags(nil)
-	flag.Set("logtostderr", "true")
-	flag.Parse()
+	logs.InitLogs()
+	defer logs.FlushLogs()
+
+	logOptions := logs.NewOptions()
+	logOptions.AddFlags(pflag.CommandLine)
+
+	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
+	pflag.Set("logtostderr", "true")
```

Comment:
Do we still need to set `logtostderr`? It might be the default in klog/v2 now.

If we can simplify it, now would be a good time.

Reply:
The flag is now called `alsologtostderr` - I doubt that this `pflag.Set` does anything :grin: